presenters/booking_presenter.rb

```diff
@@ -0,0 +1,52 @@
+class BookingPresenter
+  TICKETS_COUNT_LIMIT = 5
+
+  attr_reader :first_class_available_seats, :second_class_available_seats,
+              :first_class_price, :second_class_price
+
+  def initialize(params)
+    @params     = params
+    @route      = Route.find_by_id(@params[:route_id])
+    @department = City.find_by_id(@route&.start_id)
+    @arrival    = City.find_by_id(@route&.destination_id)
+    @train      = Train.find_by_id(@route&.train_id)
+    @config     = Config.first
+
+    @first_class_available_seats, @second_class_available_seats = get_available_seats
```

Comment:
Why do you need that in constructor? What is I were to never use these methods? That's wasted computational time.
How about something like this
```ruby
def first_class_available_seats
  @first_class_available_seats ||= get_available_seats[:firsts_class]
end

def second_class_available_seats
  @second_class_available_seats ||= get_available_seats[:second_class]
end
```